simple_make.sh

```diff
@@ -19,7 +19,8 @@ elif which dnf; then
         git qt-devel qt-doc qt-creator qt5-qtsvg opencv-devel \
         openal-soft-devel libXScrnSaver-devel qrencode-devel \
         opus-devel libvpx-devel qt5-qttools-devel glib2-devel \
-        gdk-pixbuf2-devel gtk2-devel libsodium-devel
+        gdk-pixbuf2-devel gtk2-devel libsodium-devel \
+        qt5-qtsvg-devel ffmpeg-devel
```

Comment:
https://apps.fedoraproject.org/packages/s/ffmpeg-devel ← there appears to be no `ffmpeg-devel` package for fedora?